packages/installer/deploy.sh

```diff
@@ -3,4 +3,4 @@ set -e
 
 export INTERACTIVE=true
 
-cos install --no-format $@
\ No newline at end of file
+elemental --debug --logfile /tmp/elemental.log reset $@
```

Comment:
Im not sure if this is good to have on a production package. I mean, its good for testing, but any derivative using this is gonna get a lot of logs which makes no sense to have.

Reply:
Yes, to remove that we need to also modify our tests. However we are documenting everywhere to use `elemental` not its backward compatibility wrappers `cos-*`, so only those using these wrappers will face this logs.

We can remove `--logfile` options from wrappers but then we need to start testing in our CI using `elemental` or to embed this config with some env var. I'd suggest to do it in follow up PR so we prior backward compatibility and reduced impact on CI tests first.